fluss-server/src/test/java/com/alibaba/fluss/server/replica/ReplicaTest.java

```diff
@@ -101,7 +101,13 @@ void testMakeLeader() throws Exception {
         assertThat(kvReplica.getLogTablet()).isNotNull();
         makeKvReplicaAsLeader(kvReplica);
         assertThat(kvReplica.getLogTablet()).isNotNull();
-        // TODO get kv is true.
+        assertThat(kvReplica.getKvTablet()).isNotNull();
+
+        // make as leader with leader epoch change
+        int expectedLeaderEpoch = 1;
+        makeKvReplicaAsLeader(kvReplica, expectedLeaderEpoch);
+        assertThat(kvReplica.getLeaderEpoch()).isEqualTo(expectedLeaderEpoch);
+        assertThat(kvReplica.getLogTablet()).isNotNull();
```

Comment:
Could you add a test that can reproduce the issue problem? Currently, it passes for old code.